darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -2093,6 +2094,201 @@ def test_catboost_model_with_eval_set(self, lgb_fit_patch):
         assert lgb_fit_patch.call_args[1]["eval_set"] is not None
         assert lgb_fit_patch.call_args[1]["early_stopping_rounds"] == 2
 
+    def test_fit_predict_with_categorical_features(self):
```

Comment:
Could we maybe expand this test a bit so that we can test that the categorical variables actually improve the forecast?

Reply:
Yes! See `test_quality_forecast_with_categorical_covariates`